Sources/StreamChatUI/ChatChannel/ChatChannelVC.swift

```diff
@@ -254,7 +254,10 @@ open class ChatChannelVC: _ViewController,
 
     /// Marks the channel read and updates the UI optimistically.
     public func markRead() {
-        channelController.markRead { [weak self] _ in
+        channelController.markRead { [weak self] error in
+            if error == nil {
+                self?.hasSeenLastMessage = false
+            }
```

Comment:
This is the actual fix.
After marking the channel as read, we want to reset whether the last cell has been seen or not. 
When it is seen again, this value will be marked as true